tests/device_tests/test_busy_state.py

```diff
@@ -69,11 +69,11 @@ def test_busy_expiry(client: Client):
     _assert_busy(client, True)
 
     # Hasn't expired yet.
-    time.sleep(1.4)
+    time.sleep(0.1)
     _assert_busy(client, True)
 
     # Wait for it to expire. Add 400ms tolerance to account for CI slowness.
-    time.sleep(0.5)
```

Comment:
please edit the comments too

Reply:
af90ae11ffeaba4c2767230ea851ea157d1e186e